xml/System/TimeProvider.xml

```diff
@@ -242,9 +257,9 @@
         <ReturnType>System.Int64</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the frequency of <see cref="M:System.TimeProvider.GetTimestamp" /> of high-frequency value per second.</summary>
```

Comment:
This description doesn't make sense to me.

Reply:
CC. @tarekgh

Should this be something more like the following?
```suggestion
        <summary>Gets the frequency of <see cref="M:System.TimeProvider.GetTimestamp" /> as the number of ticks per second.</summary>
```

Reply:
Yes, @tannergooding suggestion is correct and good.